src/gwkokab/models/hybrids/_bp2pmultispinfull.py

```
@@ -0,0 +1,688 @@
+        "cos_tilt_2_loc_bpl2": constraints.interval(-1.0, 1.0),
+        "cos_tilt_2_loc_n1": constraints.interval(-1.0, 1.0),
+        "cos_tilt_2_loc_n2": constraints.interval(-1.0, 1.0),
+        "cos_tilt_2_scale_bpl1": constraints.positive,
+        "cos_tilt_2_scale_bpl2": constraints.positive,
+        "cos_tilt_2_scale_n1": constraints.positive,
+        "cos_tilt_2_scale_n2": constraints.positive,
+        "cos_tilt_zeta_bpl1": constraints.unit_interval,
+        "cos_tilt_zeta_bpl2": constraints.unit_interval,
+        "cos_tilt_zeta_n1": constraints.unit_interval,
+        "cos_tilt_zeta_n2": constraints.unit_interval,
+    }
```

Ruff (0.14.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)